### DISCUSSION TITLE
Cosmos GUI

### DISCUSSION THREAD

**Discussion by Tsegab432** [CosmosOS/Cosmos#2888] [Q&A] [answered]
Is the GUI from the cosmos removed?

**Comment by ascpixi** [comment, marked as answer]
Cosmos only provides a base kernel for you to build upon - it never included a GUI of any kind. Usually, kernels do not have GUIs (although most authors extending the Cosmos kernel choose to embed the GUI directly inside their kernel, never really going into user-mode).
